plugins/woocommerce-blocks/tests/e2e/tests/add-to-cart-form/add-to-cart-form.block_theme.spec.ts

```diff
@@ -42,15 +43,23 @@ class BlockUtils {
 		this.page = page;
 	}
 
-	async configureSingleProductBlock() {
+	async configureSingleProductBlock( name?: string ) {
```

Comment:
Not a blocker but this function name is kinda confusing. It took me a bit to figure out what it was actually doing.
Seems like it selects a product for the single product block. Maybe just a comment suffices?

Also I guess it always picked the first one, maybe this should be two different functions: `addFirstProductToSingleProductBlock` and `addProductToSingleProductBlockWithName`

Reply:
Since this is a private function for a specific E2E test, it seems reasonable to add a comment to make the logic more explicit. However, if you feel it is necessary to create two separate functions, I'm happy to do so.